src/components/views/auth/RegistrationForm.js

```diff
@@ -270,12 +271,23 @@ module.exports = React.createClass({
                     }
                     const { scorePassword } = await import('../../../utils/PasswordScorer');
                     const complexity = scorePassword(value);
+                    const unsafe = complexity.score < PASSWORD_MIN_SCORE;
+                    const allowUnsafe = SdkConfig.get()["dangerously_allow_unsafe_and_insecure_passwords"];
```

Comment:
I thought we preferred JS casing for new config keys, like `dangerouslyAllow...`? It's all a bit of mess, so hard to feel too strongly at this point...

Reply:
In this case I think I prefer underscores because it's easier to read (for maintainers and for people who set this for some reason), although generally I would agree that weShouldBeUsingCamelCase.